tests/std/tests/Dev11_0493504_error_category_lifetime/test.cpp

```diff
@@ -28,6 +28,17 @@ struct Global {
 
 Global global;
 
+// Also test LWG-3598: system_category().default_error_condition(0) has a generic category.
+void test_lwg_3598() {
+    error_condition cond = system_category().default_error_condition(0);
+
+    assert(cond.category() == generic_category());
+    assert(cond.message() == "success");
+    assert(cond.value() == 0);
+
+    assert(error_code() == error_condition());

```

Comment:
No change requested: Construction of these temporaries could use empty braces, but it's not worth resetting testing for such a minor style issue in test code.